api/job/dispatcher.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	DISPATCHER_PERIOD = time.Second * 5
+	DISPATCHER_PERIOD = time.Minute * 5
```

Comment:
I should probably add a comment - the 5 minutes relates to the burst read limit in dynamodb. The increase went up so high because this added the insertHook: instead of waiting every 5 seconds for a job to be picked up by the workers, the job get's immediately put into the queue. The 5 minutes is there to pick up any leftovers/jobs stuck way in the back of the queue.